Dockerfile

```diff
@@ -25,21 +25,22 @@ RUN curl -sSOL https://github.com/koalaman/shellcheck/releases/download/v0.10.0/
  && rm -fr shellcheck-* \
  && true
 
-ENV VERSION=0.1.81
+ENV VERSION=0.1.96
 
 # Install /usr/local/bin/ys (the YAMLScript interpreter binary):
-RUN curl -s https://yamlscript.org/install | BIN=1 bash \
+RUN curl -s https://yamlscript.org/install | BIN=1 VERSION=$VERSION bash \
  && curl -s https://yamlscript.org/install | BIN=1 VERSION=0.1.80 bash \
  && rm -f \
         /usr/local/bin/ys \
         /usr/local/bin/ys-0 \
         /usr/local/bin/ys-sh-$VERSION \
+        /usr/local/bin/ys-sh-0.1.80 \
  && true
 
 RUN true \
- && ln -s ys-$VERSION /usr/local/bin/ys-0.1.79 \
- && ln -s ys-$VERSION /usr/local/bin/ys-0.1.76 \
- && ln -s ys-$VERSION /usr/local/bin/ys-0.1.75 \
+ && ln -s ys-0.1.80 /usr/local/bin/ys-0.1.79 \
```

Comment:
Why are these versions hard coded?

Reply:
So the yamlscript track is pinned to a specific ys version. Even using the CLI the Makefile for `make test` ensures you are running the version the track is pinned to. Some of these versions are compatible and some are not. I symlink the compatible ones here.

When YS v1 is released things may change here, but for now it works out.